app/helpers/form_helper.rb

```diff
@@ -46,7 +46,9 @@ def checkbox_f(f, attr, options = {}, checked_value = "1", unchecked_value = "0"
   end
 
   def multiple_checkboxes(f, attr, klass, associations, options = {}, html_options = {})
-    associated_obj = klass.send(ActiveModel::Naming.plural(associations.new))
```

Comment:
The bug is basically `klass.send(ActiveModel::Naming.plural(associations.new))` when associations is `User.current.organizations` produces 'taxonomies'